risc0/zkvm/src/guest/mod.rs

```diff
@@ -63,6 +63,8 @@ pub mod sha;
 
 use core::{arch::asm, mem, ptr};
 
+// Empty import to get the externc_libm symbols linked into the final linker run.
+use externc_libm::math::fminf;
```

Comment:
Ugh , there's got to be a better way to do this but I don't know what it is. :)

Reply:
Found it!
```rust
extern crate externc_libm;
```